telegram/constants.py

```diff
@@ -25,7 +25,9 @@
 those classes.
 
 .. versionchanged:: 20.0
-    Since v20.0, most of the constants in this module are grouped into enums.
+
+    * Most of the constants in this module are grouped into enums.
+    * ``str(<Enum>)`` now returns the value of the enum instead of the name of the enum.
```

Comment:
```suggestion
Most of the constants in this module are grouped into enums.
``str(<Enum>)`` now the value of the enum, while ``repr(<Enum>)`` returns thef the name of the enum.
```

differences between pre-releases don't need to be documented

Reply:
oh right I forgot, Enum's have been here for so long already